dataconnect/dataconnect/connector/queries.gql

```diff
@@ -0,0 +1,576 @@
+# List subset of fields for movies
+query ListMovies @auth(level: PUBLIC) {
+  movies {
+    id
+    title
+    imageUrl
+    releaseYear
+    genre
+    rating
+    tags
+    description
+  }
+}
+
+
+# List subset of fields for users
+query ListUsers @auth(level: PUBLIC) {
+  users {
+    id
+    username
+    favoriteActors_on_user {
```

Comment:
Emm, would it make sense to add an alias `favoriteActors: favoriteActors_on_user` for readability?

I think we should recommend that.